services/core-api/app/api/securities/models/bond.py

```diff
@@ -44,6 +48,17 @@ class _ModelSchema(Base._ModelSchema):
     def __repr__(self):
         return '<Bond %r>' % self.bond_guid
 
+    def save_bond_history(self):
+        bond_json = marshal(self, BOND)
```

Comment:
can this use BondHistory._schema().dumps, this is currently using two different librarys for dumping and loading. Which is how the api works, but can you see if that's an easy change?

Reply:
I will try that